addons/mail/static/src/components/chatter/chatter_suggested_recipient_tests.js

```diff
@@ -374,6 +374,45 @@ QUnit.test("suggested recipients list display 3 suggested recipient and 'show mo
     );
 });
 
+QUnit.test("suggested recipients should not be notified when posting an internal note", async function (assert) {
+    assert.expect(1);
+
+    this.data['res.partner'].records.push({
+        display_name: "John Jane",
+        email: "john@jane.be",
+        id: 100,
+    });
+    this.data['res.fake'].records.push({
+        id: 10,
+        partner_ids: [100],
+    });
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.model === 'res.fake' && args.method === 'message_post') {
+                assert.strictEqual(
+                    args.kwargs.partner_ids.length,
+                    0,
+                    'message_post should not contain partners for lognote'
```

Comment:
My suggestion also included this text:

"message_post should not contain suggested recipients when posting an internal note"